src/Tizen.CoreUI/src/CoreUI.Canvas/AnimationTypes.cs

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 by its authors. See AUTHORS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma warning disable CS1591
```

Comment:
You don't have to include this pragma in every file, because the CS1591 warning is related to undocumented public types and methods, which you have already done. I suggest using it only if it is necessary.